lib/plugins/aws/package/compile/events/schedule.js

```diff
@@ -68,16 +69,60 @@ class AwsCompileScheduledEvents {
               },
               required: ['inputTemplate'],
               additionalProperties: false,
+            }
+          },
+          required: ['rate'],
+          additionalProperties: false,
+        },
+        {
+          type: 'object',
+          properties: {
+            rate: {
+              type: 'array',
+              minItems: 1,
+              items: {
+                type: 'string',
+                pattern: scheduleSyntax,
+              },
+            },
+            enabled: { type: 'boolean' },
+            name: { type: 'string', minLength: 1, maxLength: 64, pattern: '[\\.\\-_A-Za-z0-9]+' },
+            description: { type: 'string', maxLength: 512 },
+            input: {
+              anyOf: [
+                { type: 'string', maxLength: 8192 },
+                {
+                  type: 'object',
+                  oneOf: [
+                    {
+                      properties: {
+                        body: { type: 'string', maxLength: 8192 },
+                      },
+                      required: ['body'],
+                      additionalProperties: false,
+                    },
+                    {
+                      not: {
+                        required: ['body'],
+                      },
+                    },
+                  ],
+                },
+              ],
             },
             method: {
               type: 'string',
-              enum: ['eventBus', 'scheduler'],
+              enum: ['eventBus'],
             },
             roleArn: {
               type: 'string',
             },
+            timezone: {
+              type: 'string',
+              enum: TIME_ZONES
```

Comment:
Does CloudFormation check this? If so, the enum is probably not necessary since CF will return the error

Reply:
As per cloudformation's spec, there's no enum defined, but it throws an error and the whole stack has to be roll backed. Seen it happen, hence added it here.

Reply:
Even if Cloudformation was validating it, doing this would also generate the types properly and save people time that was spent on packaging.